eZ/Publish/Core/Repository/SiteAccessAware/URLAliasService.php

```diff
@@ -96,4 +96,14 @@ public function load($id)
     {
         return $this->service->load($id);
     }
+
```

Comment:
Can we add @inheritdoc or docblock here and below?

Reply:
> Can we add `@inheritdoc` or docblock here and below?

This is not consistent with the rest of the file. Moreover we had a lot of discussions about `@inheritdoc` and is seems we tend to [misuse it](https://docs.phpdoc.org/guides/inheritance.html). Also, it is not needed since most of explored tools properly show parent doc (as required by the PHPDoc Standard).

I add it for consistency only, in classes which already have it added.